.github/workflows/deployFreighterApiProduction.yml

```diff
@@ -0,0 +1,49 @@
+name: Deploy @stellar/freighter-api
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: Release Number
+        default: "2.0.0"
+        required: true
+jobs:
+  bump-version:
+    name: Checkout code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: release/${{ github.event.inputs.release }}
+      - name: Build package
+        uses: actions/setup-node@v4
+        with:
+          node-version: "21"
+      - name: Version npm package
+        run: |
+          cd @stellar/freighter-api
+          yarn config set version-tag-prefix "@stellar/freighter-api@"
+          yarn config set version-git-message "@stellar/freighter-api: bumping version to %s"
```

Comment:
`yarn version` will automatically bump the version, tag, and commit the change. These 2 configs help us customize the tag name and the commit message

Once versioned, we just need to open a PR with the committed changes